src/backend/distributed/commands/common.c

```diff
@@ -272,3 +274,67 @@ PreprocessDropDistributedObjectStmt(Node *node, const char *queryString,
 
 	return NodeDDLTaskList(NON_COORDINATOR_NODES, commands);
 }
+
+
+/*
+ * DropTextSearchDictObjectAddress returns object address for the drop TextSearchDict stmt.
+ */
+ObjectAddress
+DropTextSearchDictObjectAddress(Node *node, bool missing_ok)
+{
+	DropStmt *stmt = castNode(DropStmt, node);
+
+	ObjectAddress objectAddress = { 0 };
+
+	List *objName = NIL;
+
+	foreach_ptr(objName, stmt->objects)
+	{
+		char *schemaName = NULL;
+		char *tsdictName = NULL;
+		DeconstructQualifiedName(objName, &schemaName, &tsdictName);
+
+		if (!schemaName)
```

Comment:
I'm a bit confused with this code. `get_ts_dict_oid` already calls `DeconstructQualifiedName`, why do we need to call again?

In other words, isn't `get_ts_dict_oid(objName, true)` already enough for us to calculate the objectAddress?

Reply:
Same applies for `DropTextSearchConfigObjectAddress`